mod/primitives/pkg/net/jwt/jwt_test.go

```diff
@@ -105,95 +102,54 @@ func TestSecretString(t *testing.T) {
 
 	for _, tt := range tests {
 		t.Run(tt.name, func(t *testing.T) {
-			if got := tt.secret.String(); got != tt.want {
-				t.Errorf("Secret.String() = %v, want %v", got, tt.want)
-			}
+			require.Equal(t, tt.want, tt.secret.String(), "Secret.String() mismatch")
```

Comment:
If both are equal, then why mismatch message ? Kindly check the condition once, it should be NotEqual.